modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -161,7 +162,7 @@ public interface {{classname}} {
             {{#responses}}
             @ApiResponse(responseCode = "{{{code}}}", description = "{{{message}}}"{{#baseType}}, content = {
                 {{#produces}}
-                @Content(mediaType = "{{{mediaType}}}", schema = @Schema(implementation = {{{baseType}}}.class)){{^-last}},{{/-last}}
+                @Content(mediaType = "{{{mediaType}}}", {{#isArray}}array = @ArraySchema( {{/isArray}}schema = @Schema(implementation = {{{baseType}}}.class){{#isArray}}){{/isArray}}){{^-last}},{{/-last}}
```

Comment:
Does this require any version checks for Swagger? Or it should have always been there?

Reply:
seems like it's in all versions of v3 package

https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-annotations/src/main/java/io/swagger/v3/oas/annotations/media/Content.java#L70

Reply:
The while section is wrapped in  a `{{#swagger2AnnotationLibrary}}` Block. So it is only in effect with Swagger v2.x / OAS v3.